src/validate_pyproject/formats.py

```diff
@@ -378,7 +378,25 @@ def int(value: builtins.int) -> bool:
     return -(2**63) <= value < 2**63
 
 
-def SPDX(value: str) -> bool:
-    """Should validate eventually"""
-    # TODO: validate conditional to the presence of (the right version) of packaging
-    return True
+try:
+    from packaging import licenses as _licenses
+
+    def SPDX(value: str) -> bool:
+        """See :ref:`PyPA's license specifiers <pypa:#license>`
```

Comment:
This cross-reference seems to be undefined.

Maybe `<pypa:core-metadata-license-expression>`? (untested)
Something else?